python/adelphi/tests/integration/resources/nb-schemas/2.1.22.yaml

```diff
@@ -10,7 +10,7 @@ blocks:
     cl: LOCAL_QUORUM
   statements:
   - rampup-insert: insert into "testkeyspace"."testtable" ("key1","key2","value")
-      values ({key1_seq},{key2_dist},{value_dist})
+      values ({key1_seq},{key2_seq},{value_dist})
```

Comment:
An interesting fallout from the integration test fixes above.  The use of a compound primary key in our test schema means that we identify both key1 and key2 as in the primary key and therefore both should use the seq rather than the dist.  Technically this is overkill; obviously a seq from a single column would guarantee uniqueness (meaning the original schema is certainly correct).  But this version is not wrong (it's just excessively unique) and it does keep the logic in the impl very straightforward (if primary key use seq, otherwise use dist)... so I updated the reference schemas instead.